lib/v2/lovelive-anime/schedules.js

```diff
@@ -0,0 +1,56 @@
+const got = require('@/utils/got');
+const path = require('path');
+const { art } = require('@/utils/render');
+const renderDescription = (desc) => art(path.join(__dirname, 'templates/scheduleDesc.art'), desc);
+const dayjs = require('dayjs');
+const utc = require('dayjs/plugin/utc');
+const timezone = require('dayjs/plugin/timezone');
+
+module.exports = async (ctx) => {
+    dayjs.extend(utc);
+    dayjs.extend(timezone);
+    const serie = ctx.params.serie;
+    const category = ctx.params.category;
+    const rootUrl = `https://www.lovelive-anime.jp/common/api/calendar_list.php`;
+    const nowTime = dayjs();
+    const dataPara = {
+        year: nowTime.year(),
+        month: nowTime.format('MM'),
+    };
+    if (serie && 'all' !== serie) {
+        dataPara.series = [serie];
+    }
+    if (category && 'all' !== category) {
+        dataPara.category = [category];
+    }
+    const response = await got(`${rootUrl}?site=jp&ip=lovelive&data=${JSON.stringify(dataPara)}`);
+
+    const items = response.data.data.schedule_list.map((item) => {
+        const link = item.url.select_url;
+        const title = item.title;
+        const category = item.categories.name;
+        const eventStartDate = dayjs.unix(item.event_startdate).tz('Asia/Tokyo').format('YYYY-MM-DD HH:mm:ss JST');
+        const eventEndDate = dayjs.unix(item.event_enddate).tz('Asia/Tokyo').format('YYYY-MM-DD HH:mm:ss JST');
+
+        return {
+            link,
+            pubDate: '',
```

Comment:
Maybe set `pubDate` to eventStartDate?

Reply:
The `eventStartDate` in the website api is not recommended for `pubDate`, the data obtained by the API has more complex time attributes, so I simply leave the pubDate empty.

Reply:
The data from the API is not complex at all. They are simply unix timestamps.
`pubDate: parseDate(item.created, 'X')` will do the trick. `created` is more preferred.
```json
{
  "event_enddate": 1668956340,
  "reset": 1663223829626,
  "updated_user": "脇黒丸 翔太",
  "lng": "jp",
  "startdate": 1662044400,
  "_p_key": "lovelive_jp",
  "site": "jp",
  "url": {
    "select_transition": "2",
    "select_url": "https://www.lovelive-anime.jp/yuigaoka/live/sp_3rdlive/ticket.php"
  },
  "series": [
    {
      "name": "スーパースター!!",
      "code": "superstar"
    }
  ],
  "_id": "13",
  "id": "13",
  "created": 1662094082,
  "ip": "lovelive",
  "event_startdate": 1664334000,
  "start_endflg": "1",
  "created_user": "脇黒丸 翔太",
  "updated": 1663223829,
  "enddate": 32503647540,
  "publish_status": "publish",
  "categories": {
    "name": "ライブ",
    "code": "live"
  },
  "delflg": "0",
  "event_dspdate": "＜最速先行抽選＞申込受付期間：2022年9月28日～2022年11月20日（TVアニメ2期 Blu-ray第1巻封入）",
  "period_label": "受付中",
  "title": "【チケット情報】ラブライブ！スーパースター!! Liella! 3rd LoveLive! Tour ～WE WILL!!～＜千葉公演＞"
}
```

Reply:
I just discovered the json has a key called `start_endflg`, it identifies whether to use `event_startdate` or `event_enddate`.
I have committed the code related to `pubDate`